### ISSUE TITLE
Cannot set the useExecutor value within the job files

### ISSUE THREAD

**Issue by devangorder** [azkaban/azkaban#656]
It appears the useExecutor override value can only be set within the flow params. Can this be integrated into the jobfile itself, so when I load a project zip, certain jobs are always configured to run on a specific executor by default?

**Comment by vermamitesh87** [comment]
@devangorder I have gone through #667 . Were you able to force execution of a job to a particular executor ?

**Comment by devangorder** [comment]
Yes, by setting the useExecutor flow override manually when launching a flow, or through the API when scheduling a flow. We still cannot pull this value from the job files.

**Comment by zixuedanxin** [comment]
> Yes, by setting the useExecutor flow override manually when launching a flow, or through the API when scheduling a flow. We still cannot pull this value from the job files.

how can I set the "useExecutor" when I   scheduling a flow through the API. Do you have a demo or example.
I tried like this:

       cron = flow_cron  # prj_cron.get(self.prj_name + self.flowId, None)
        if check_cron(cron):
            data = {
                'session.id': self.cookies_fetcher.get_session_id(),
                'ajax': u'scheduleCronFlow',
                'projectName': self.prj_name,
                'flow': self.flowId,
                'flowParameters': {
                    'useExecutor': '2'
                },
                'failureAction': 'finishPossible',  # finishCurrent, cancelImmediatel useExecutor (需要配合hostname)
                'cronExpression': cron
            }

            response = requests.post(
                azkaban_url + '/schedule',
                data=data
            )

But it does't work
